sdk/containerregistry/azure-containers-containerregistry/src/main/java/com/azure/containers/containerregistry/models/ManifestMediaType.java

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.containers.containerregistry.models;
+
+import com.azure.core.util.ExpandableStringEnum;
+
+/**
+ * Manifest media type.
+ */
+public class ManifestMediaType extends ExpandableStringEnum<ManifestMediaType> {
+
+    /**
+     * Docker v2 manifest content type.
+     */
+    public static final ManifestMediaType DOCKER_MANIFEST = fromString("application/vnd.docker.distribution.manifest.v2+json");
```

Comment:
Is this being copied out of code generated implementation?

Reply:
nope, this enum is populated from docker v2 documentation.
swagger does not define all supported content types.